qa/workunits/rados/test_dedup_tool.sh

```diff
@@ -306,10 +306,57 @@ function test_dedup_object()
   $RADOS_TOOL -p $POOL rm bar
 }
 
+function test_sample_dedup()
+{
+  CHUNK_POOL=dedup_chunk_pool
+  run_expect_succ "$CEPH_TOOL" osd pool create "$POOL" 8
+  run_expect_succ "$CEPH_TOOL" osd pool create "$CHUNK_POOL" 8
+  run_expect_succ "$CEPH_TOOL" osd pool set "$POOL" dedup_tier "$CHUNK_POOL"
+  run_expect_succ "$CEPH_TOOL" osd pool set "$POOL" dedup_chunk_algorithm fastcdc
+  run_expect_succ "$CEPH_TOOL" osd pool set "$POOL" dedup_cdc_chunk_size 8192
+  run_expect_succ "$CEPH_TOOL" osd pool set "$POOL" fingerprint_algorithm sha1
+  CONTENT="There hiHI"
+
+  echo $CONTENT > foo
+  $RADOS_TOOL -p $POOL put foo1 ./foo
+  sleep 2
+
+  $RADOS_TOOL -p $POOL put foo2 ./foo
+  sleep 2
+
```

Comment:
make more than 10 objects, then examine the object to check dedup work is correct.